Automattic/pocket-casts-ios
# Remove iOS 14 support

As we develop more new features using SwiftUI, iOS 14 has proven to be quite old and requires a lot of work for simple changes in UI.

iOS 14 user base is quite small so we're removing support for it.

## To test

1. Do a thorough code review.
2. Run the app and make sure everything is fine (including widgets)

## Checklist
